TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -1016,3 +1047,96 @@ bool Moveable::MeshExists(int index) const
 
 	return true;
 }
+
+//Attach camera and camera target to a mesh of an object.
+void  Moveable::AttachCamera(short camMeshId, short targetMeshId)
+{
+	ObjCamera(m_item, camMeshId, m_item, targetMeshId, true);
+}
+
+//Lara will undraw her weapons if they are drawn, throw away flare if she helds one in her hand.
+void   Moveable::UndrawWeapons()
+{
+	m_item->IsLara();
```

Comment:
These calls look like they do nothing; please remove them, or if they will end up getting used once we have a Lara class, comment them out and add a `//todo` comment

Reply:
fixed